tls/s2n_kem.c

```diff
@@ -102,6 +103,24 @@ const struct s2n_iana_to_kem kem_mapping[2] = {
         }
 };
 
+/* Specific assignments of KEM group IDs and names have not yet been
+ * published in an RFC (or draft). For IDs, there is consensus to use
+ * values in the proposed reserved range defined in
+ * https://tools.ietf.org/html/draft-stebila-tls-hybrid-design */
```

Comment:
Is there consensus on using these values for SIKE and BIKE? Will this interoperate with other implementations? Also this feels like something we should call out in the PQ readme.

Reply:
These values will interoperate with other implementations. There is nothing officially published, but there is consensus within the community. See:
* https://github.com/open-quantum-safe/openssl/blob/2faa797e36a697be8199da20322f58b85aacf86d/oqs-template/generate.yml#L2
* https://docs.google.com/spreadsheets/d/12YarzaNv3XQNLnvDsWLlRKwtZFhRrDdWf36YlzwrPeg/edit#gid=0

I can update the comment with that Google doc reference.